schemas/2019-05-15/Microsoft.Kusto.json

```diff
@@ -0,0 +1,483 @@
+﻿{
```

Comment:
See https://github.com/Azure/azure-resource-manager-schemas#hints - you need to add references to this schema to the top-level schemas. Otherwise, the changes won't be visible to anyone

Reply:
Thanks, please see second commit: add_reference_to_top_level_schemas